diff --git a/dist/node/chunks/dep-M1IYMR16.js b/dist/node/chunks/dep-M1IYMR16.js
index 6de6e61108137b1c9491e37dad7834815d247ed2..485e366fb62bc380a15e6a5c1a3114488f130019 100644
--- a/dist/node/chunks/dep-M1IYMR16.js
+++ b/dist/node/chunks/dep-M1IYMR16.js
@@ -13678,6 +13678,7 @@ function manifestPlugin() {
       manifest: {},
       outputCount: 0,
       reset() {
+        this.manifest = {}
         this.outputCount = 0;
       }
     };
